plugins/SitesManager/API.php

```diff
@@ -1346,6 +1358,68 @@ public function setDefaultTimezone($defaultTimezone)
         return true;
     }
 
+    /**
+     * Sets global query parameter exclusion based on the specified exclusion type.
+     *
+     * @param string $exclusionType The type of query param exclusion, must be of the following:
+     *  - no_exclusions
+     *  - common_pii_exclusions
+     *  - custom_exclusions
+     * @param string|null $queryParamsToExclude (Optional) List of query parameters to exclude when $exclusionType is 'custom'.
+     *                                         Ignored if $exclusionType is not 'custom'.
+     * @return void
+     * @throws InvalidArgumentException
+     */
+    public function setGlobalQueryParamExclusion(string $exclusionType, ?string $queryParamsToExclude = null): void
+    {
+        Piwik::checkUserHasSuperUserAccess();
+
+        $whiteListValidator = new WhitelistedValue(SitesManager::URL_PARAM_EXCLUSION_TYPES);
+        $whiteListValidator->validate($exclusionType);
+
+        if ($exclusionType === SitesManager::URL_PARAM_EXCLUSION_TYPE_NAME_CUSTOM_EXCLUSIONS && empty($queryParamsToExclude)) {
+            throw new Exception($this->translator->translate('ExceptionEmptyQueryParamsForCustomType'));
+        }
+
+        if ($exclusionType !== SitesManager::URL_PARAM_EXCLUSION_TYPE_NAME_CUSTOM_EXCLUSIONS && !empty($queryParamsToExclude)) {
+            throw new Exception($this->translator->translate('ExceptionNonEmptyQueryParamsForNonCustomType'));
+        }
```

Comment:
```suggestion
        if ($exclusionType === SitesManager::URL_PARAM_EXCLUSION_TYPE_NAME_CUSTOM_EXCLUSIONS && empty($queryParamsToExclude)) {
            throw new Exception($this->translator->translate('SitesManager_ExceptionEmptyQueryParamsForCustomType'));
        }

        if ($exclusionType !== SitesManager::URL_PARAM_EXCLUSION_TYPE_NAME_CUSTOM_EXCLUSIONS && !empty($queryParamsToExclude)) {
            throw new Exception($this->translator->translate('SitesManager_ExceptionNonEmptyQueryParamsForNonCustomType'));
        }
```

The translations here need the plugin name to be translated :see_no_evil: